.github/workflows/approve-readme.yaml

```diff
@@ -0,0 +1,54 @@
+on:
```

Comment:
Should this file be part of this PR?

Reply:
No, it's part of the generated changes by running the synth tool. It should be removed before we merge it.